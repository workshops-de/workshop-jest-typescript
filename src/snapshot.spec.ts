describe('Snapshot', () => {
  describe('When the application starts', () => {
    it('displays the current version', () => {
      const version = '1.1';
      const mockTemplate = `<header>Version ${version}</header>`;

      expect(mockTemplate).toMatchSnapshot();
    });
  });
});
