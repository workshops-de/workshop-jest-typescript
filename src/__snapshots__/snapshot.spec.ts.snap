// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot When the application starts displays the current version 1`] = `"<header>Version 1.1</header>"`;
